libs/agno/agno/os/utils.py

```diff
@@ -495,3 +508,31 @@ def collect_mcp_tools_from_workflow_step(step: Any, mcp_tools: List[Any]) -> Non
     elif isinstance(step, Workflow):
         # Nested workflow
         collect_mcp_tools_from_workflow(step, mcp_tools)
+
+
+def input_content_string(input_content: Union[str, Dict[str, Any], List[Any], BaseModel]) -> str:
+    """Convert input_content to a string representation.
+    
+    Handles both serialized (dict) and live (object) formats.
+    """
```

Comment:
```suggestion
def stringify_input_content(input_content: Union[str, Dict[str, Any], List[Any], BaseModel]) -> str:
    """Convert any given input_content into its string representation.
    
    This handles both serialized (dict) and live (object) input_content formats.
    """
```

If we accept this we need to also update the call on line 68

Reply:
yeah happy with the name change